flink-clients/src/test/java/org/apache/flink/client/cli/CliFrontendInfoTest.java

```diff
@@ -45,7 +45,7 @@ public void testMissingOption() throws Exception {
 		CliFrontend testFrontend = new CliFrontend(
 			configuration,
 			Collections.singletonList(getCli()));
-		testFrontend.cancel(parameters);
```

Comment:
This used to be `cancel`, but i thought it was a typo mistake, so i changed it to `info`.